README.md

```diff
@@ -109,6 +109,7 @@ The **API endpoint** can be used to:
 - `[BETA]` Configure the data tree of the guest-facing metadata service. The
   service is only available to the guest if this resource is configured.
 - Add a [vsock socket](docs/vsock.md) to the microVM.
+- Add a [entropy device](docs/entropy.md) to the microVM.
```

Comment:
Might be slightly off topic, but don't we need to update the changelog?

Reply:
We do! I was planning to do this once we are ready to merge this to `main`. There might be more functionality coming in the feature branch before that.